### ISSUE TITLE
`@time_imports` to show total time as well

### ISSUE THREAD

**Issue by baggepinnen** [JuliaLang/julia#46850]
I frequenctly find myself calling
```julia
@time_imports using SlowPackage
```
followed by a restart of julia and then
```julia
@time using SlowPackage
```
to get the total time. It would be nice if `@time_imports` would print the total time as well.

**Comment by baggepinnen** [comment]
```
@time @time_imports using SlowPackage
```
is a valid approach to get both at the same time, but it requires you to remember to use both

**Comment by IanButterworth** [comment]
Yeah, it should do this automatically. The first version before release did, but that was because of a bug in code loading that made the loading partially nested, but that was fixed.
